assets/src/components/detours/diversionPage.tsx

```diff
@@ -276,6 +276,21 @@ export const DiversionPage = ({
             },
           }) ? (
             <ActivateDetour.Modal
+              nextStepLabel={
+                (snapshot.matches({
+                  "Detour Drawing": {
+                    "Share Detour": { Activating: "Selecting Duration" },
+                  },
+                }) &&
+                  "Select Duration") ||
+                (snapshot.matches({
+                  "Detour Drawing": {
+                    "Share Detour": { Activating: "Selecting Reason" },
+                  },
+                }) &&
+                  "Select Reason") ||
```

Comment:
non-blocking: Would these element ids be better as past tense, given that when the next button is clicked, the action already happened?
```suggestion
                  "Selected Duration") ||
                (snapshot.matches({
                  "Detour Drawing": {
                    "Share Detour": { Activating: "Selecting Reason" },
                  },
                }) &&
                  "Selected Reason") ||
```

Reply:
1. These labels can be changed in fullstory UI if desired by our team
2. I think given how this appears in fullstory[^1], it's more correct to use present tense. I could alternatively name these `Confirm Reason` or `Confirm Duration`

[^1]: 
![image](https://github.com/user-attachments/assets/0975edfa-fc98-4bac-8957-9672f320d0f4)

Reply:
```suggestion
                  "Confirm Duration") ||
                (snapshot.matches({
                  "Detour Drawing": {
                    "Share Detour": { Activating: "Selecting Reason" },
                  },
                }) &&
                  "Confirm Reason") ||
```

Reply:
Got it! I do prefer "Confirm" then (just so in the future we don't think these actions are attached to the radio buttons, which would be a different meaning of "Select Reason")